submarine-workbench/interpreter/spark-interpreter/pom.xml

```diff
@@ -175,6 +175,7 @@
       <groupId>org.apache.zeppelin</groupId>
       <artifactId>zeppelin-spark-dependencies</artifactId>
       <version>${zeppelin.version}</version>
+      <!--
       <exclusions>
```

Comment:
@hhhizzz Please help review this code, Thanks!

Reply:
Thanks @liuxunorg,
Just tried to figure out how to pass CI tests. I triggered travis for my repo. And it triggered this PR as well.
I'm still working on adding UT tests for this PR. This will be fixed shortly.

Reply:
Done